src/library/Uno.Themes.WinUI.Markup/Theme/TextBox.cs

```diff
@@ -0,0 +1,201 @@
+						public static ResourceValue<Brush> PlaceholderDisabled => new("FilledTextBoxPlaceholderForegroundDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForeground")]
+						public static ResourceValue<Brush> DeleteButton => new("FilledTextBoxDeleteButtonForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForegroundPointerOver")]
+						public static ResourceValue<Brush> DeleteButtonPointerOver => new("FilledTextBoxDeleteButtonForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForegroundFocused")]
+						public static ResourceValue<Brush> DeleteButtonFocused => new("FilledTextBoxDeleteButtonForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForegroundDisabled")]
+						public static ResourceValue<Brush> DeleteButtonDisabled => new("FilledTextBoxDeleteButtonForegroundDisabled", true);
+					}
+
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackground")]
+						public static ResourceValue<Brush> Default => new("FilledTextBoxBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FilledTextBoxBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackgroundFocused")]
+						public static ResourceValue<Brush> Focused => new("FilledTextBoxBackgroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledTextBoxBackgroundDisabled", true);
+
+					}
+
+					public static class BorderBrush
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrush")]
+						public static ResourceValue<Brush> Default => new("FilledTextBoxBorderBrush", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrushPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FilledTextBoxBorderBrushPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrushFocused")]
+						public static ResourceValue<Brush> Focused => new("FilledTextBoxBorderBrushFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrushDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledTextBoxBorderBrushDisabled", true);
+					}
+
+					public static class BorderThickness
+					{
+						[ResourceKeyDefinition(typeof(Thickness), "FilledTextBoxBorderThicknessNormal")]
+						public static ResourceValue<Thickness> Normal => new("FilledTextBoxBorderThicknessNormal", true);
+
+						[ResourceKeyDefinition(typeof(Thickness), "FilledTextBoxBorderThicknessFocused")]
+						public static ResourceValue<Thickness> Focused => new("FilledTextBoxBorderThicknessFocused", true);
+					}
```

Comment:
Let's add the rest of the non-Brush resources here for Filled https://github.com/unoplatform/Uno.Themes/blob/7e9fdb04ea54c7a6c7927ad6ed4ab363e7b1f22d/src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml#L100-L104